src/Logging/Logging.Console/ConsoleLogService.cs

```diff
@@ -1,45 +1,27 @@
-﻿using static CodeMonkeys.Argument;
-
-using System;
-
-namespace CodeMonkeys.Logging.Console
+﻿namespace CodeMonkeys.Logging.Console
 {
-    internal sealed class ConsoleLogService : ILogService
+    /// <summary>
+    /// <see cref="IScopedLogService"/> which writes to <see cref="System.Console"/>.
+    /// </summary>
+    public sealed class ConsoleLogService : ScopedLogService<ConsoleLogOptions>
     {
-        private readonly ConsoleLogServiceProvider _provider;
-        private readonly string _context;
-
-        internal ConsoleLogService(
-            ConsoleLogServiceProvider provider, 
-            string context)
+        internal ConsoleLogService(string context)
+            : base(context)
         {
-            _provider = provider;
-            _context = context;
+            if (Options.ColorizeOutput)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)